integreat_cms/cms/templates/authentication/_base.html

```diff
@@ -26,6 +26,22 @@ <h1 class="heading text-center mb-4">
                     {{ COMPANY }}
                 </a>
             </p>
+            <p class="text-center text-gray-400">
+                <a href="{{ PRIVACY_POLICY_URL }}"
+                   class="text-gray-600 hover:text-gray-800"
+                   target="_blank"
+                   rel="noopener noreferrer">
+                    {% trans "Privacy policy" %}
+                </a>
+            </p>
+            <p class="text-center text-gray-400">
+                <a href="{{ IMPRINT_URL }}"
+                   class="text-gray-600 hover:text-gray-800"
+                   target="_blank"
+                   rel="noopener noreferrer">
+                    {% trans "Imprint" %}
+                </a>
+            </p>
```

Comment:
```suggestion
            <div class="mt-2 flex justify-center space-x-4 text-sm">
                <a href="{{ PRIVACY_POLICY_URL }}"
                   class=" text-gray-400 hover:text-gray-800"
                   target="_blank"
                   rel="noopener noreferrer">
                    {% trans "Privacy policy" %}
                </a>
                <a href="{{ IMPRINT_URL }}"
                   class="text-center text-gray-400 hover:text-gray-800"
                   target="_blank"
                   rel="noopener noreferrer">
                    {% trans "Imprint" %}
                </a>
            </div>
```
I don't think that we have to wrap the `<a>`-tag inside a `<p>`-tag in this case. Instead I'd just move the text color directly onto the `<a>`-tag. Also in this case, I think it looks a little better if the font is a bit smaller and if the two links are grouped together. Please feel free to dismiss the suggestion if you don't like it